logfire/_internal/integrations/openai.py

```diff
@@ -59,30 +86,30 @@ def instrument_openai_sync(logfire_openai: Logfire, openai_client: openai.OpenAI
     # WARNING: this method is vey similar to `instrument_openai_async` below, any changes here should be reflected there
     openai_client._original_request_method = original_request_method = openai_client._request  # type: ignore
 
-    def instrumented_openai_request(**kwargs: Any) -> Any:
+    def instrumented_openai_request(**kwargs: Unpack[OpenAIRequest[ResponseT, _StreamT]]) -> ResponseT | _StreamT:
         if context.get_value('suppress_instrumentation'):
             return original_request_method(**kwargs)
 
-        options: FinalRequestOptions | None = kwargs.get('options')
+        options = kwargs.get('options')
```

Comment:
`options` is mandatory, it will never be `None`.

Reply:
```suggestion
        options = kwargs['options']
```
?